torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -108,20 +108,21 @@ def test_bad_module(self):
         if self.rank != 0:
             return
         dst_worker_name = dist_utils.worker_name((self.rank + 1) % self.world_size)
+        remote_device = "{}/cpu".format(dst_worker_name)
         args = (1,)
         kwargs = dict(first_kwarg=2)
 
         with self.assertRaisesRegex(
             ValueError,
             r"Expect `module_cls\(\*args, \*\*kwargs\)` returns an instance of <class nn.Module>,",
         ):
-            RemoteModule(dst_worker_name, "cpu", BadModule, args, kwargs)
+            RemoteModule(remote_device, BadModule, args, kwargs)
```

Comment:
We should add tests for invalid remote devices, ex:

1. "trainer0/"
2. "trainer0/xyz"
3. "trainer0/cuda:0/cuda:1"
4. "/"
5. "/cuda:0"

Reply:
Thanks for the suggestion! Incorporated these test cases with `test_invalid_devices`, which has already covered case 2 you listed here.